<ui version="4.0" >
 <class>PathToolOptionWidgetBase</class>
 <widget class="QWidget" name="PathToolOptionWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="pathPage" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="stylesGroup" >
         <property name="title" >
          <string>Styles</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>3</number>
          </property>
          <property name="topMargin" >
           <number>3</number>
          </property>
          <property name="rightMargin" >
           <number>3</number>
          </property>
          <property name="bottomMargin" >
           <number>3</number>
          </property>
          <property name="horizontalSpacing" >
           <number>0</number>
          </property>
          <property name="verticalSpacing" >
           <number>3</number>
          </property>
          <item row="0" column="0" >
           <widget class="QToolButton" name="corner" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QToolButton" name="smooth" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QToolButton" name="symmetric" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QToolButton" name="lineSegment" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QToolButton" name="curveSegment" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="Line" name="line" >
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="actionsGroup" >
         <property name="title" >
          <string>Actions</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>3</number>
          </property>
          <property name="topMargin" >
           <number>3</number>
          </property>
          <property name="rightMargin" >
           <number>3</number>
          </property>
          <property name="bottomMargin" >
           <number>3</number>
          </property>
          <property name="horizontalSpacing" >
           <number>0</number>
          </property>
          <property name="verticalSpacing" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QToolButton" name="addPoint" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QToolButton" name="removePoint" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QToolButton" name="breakPoint" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QToolButton" name="breakSegment" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QToolButton" name="joinSegment" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QToolButton" name="mergePoints" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="parameterPage" >
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QStackedWidget" name="configWidget" >
         <widget class="QWidget" name="page_2" />
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="convertToPath" >
         <property name="text" >
          <string>Convert To Path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
